other/upgrade_2-1_postgresql.sql

```diff
@@ -2055,15 +2055,19 @@ CREATE INDEX {$db_prefix}ban_items_id_ban_ip ON {$db_prefix}ban_items (ip_low,ip
 --- helper function for ip convert
 /******************************************************************************/
 ---# the function migrate_inet
-CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
-AS
-$$
-BEGIN
-   RETURN (trim(val))::inet;
-EXCEPTION
-   WHEN OTHERS THEN RETURN NULL;
-END;
-$$ LANGUAGE plpgsql;
+---{
+upgrade_query("
+	CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
+	AS
+	$$
+	BEGIN
+	   RETURN (trim(val))::inet;
+	EXCEPTION
+	   WHEN OTHERS THEN RETURN NULL;
+	END;
+	$$ LANGUAGE plpgsql;"
+);
+---}
```

Comment:
This worked with the change to Subs-DB-postgresql.php

I got the upgrade working from the scratch- almost. Below are some notes:

Installation got stuck at:
```
Nov  5 13:31:55 cerebro postgres[12375]: [3-1] ERROR:  index "verstas_log_group_requests_id_member" does not exist
Nov  5 13:31:55 cerebro postgres[12375]: [3-2] STATEMENT:  DROP INDEX verstas_log_group_requests_id_member
Nov  5 13:33:42 cerebro postgres[12648]: [3-1] ERROR:  column "disregarded" does not exist
Nov  5 13:33:42 cerebro postgres[12648]: [3-2] STATEMENT:  
Nov  5 13:33:42 cerebro postgres[12648]: [3-3]          ALTER TABLE verstas_log_topics
Nov  5 13:33:42 cerebro postgres[12648]: [3-4]          RENAME disregarded TO unwatched

```
And continued only after page reload

After the database changes and hitting continue blank screen with:

`ERROR: relation "verstas_background_tasks" does not exist LINE 3: FROM verstas_background_tasks ^

`Is there a way to do the JSON step without running the whole upgrade script btw?

Then there's a slight typo in Sources/ManageSearch.php line 247
it is `                                 CREATE INDEX smf_messages_ftx ON smf_messages ` whern it should be                                  `CREATE INDEX smf_messages_ftx ON {db_prefix}messages` 

-Reko